convert/convert.go

```diff
@@ -35,9 +35,45 @@ type builder struct {
 	imports []*proto.Import
 }
 
-// Run converts a proto file to a gunk file, saving the file in the same folder
-// as the proto file.
-func Run(path string, overwrite bool) error {
+// Run converts proto files or folders to a gunk files, saving the file in
```

Comment:
nit: `s/to a gunk files, saving the file/to gunk files, saving the files/`

Reply:
Done.